Userland/Services/LaunchServer/Launcher.cpp

```diff
@@ -263,6 +277,7 @@ void Launcher::for_each_handler(String const& key, HashMap<String, String>& user
     auto user_default = user_preference.get("*");
     if (counted == 0 && user_default.has_value())
         f(get_handler_for_executable(Handler::Type::UserDefault, user_default.value()));
+    return counted;
```

Comment:
Please leave a comment somewhere that explains what the return value of the function means as I find it unintuitive.

Reply:
```
     // Return the number of times f() was called,
    // which can be used to know whether there were any handlers
```
That's used in `open_with_user_preferences` so that we don't open more than one app per file